lib/menu_printer.rb

```diff
@@ -1,35 +1,38 @@
+require_relative "font_color_and_style"
+
 class MenuPrinter
-  attr_accessor :stdout
+  attr_accessor :stdout, :colors
```

Comment:
I think the naming `colors` makes some sense, but what about `format`. Then you would have code that reads like `format.success(...`. That reads to me more like a sentence. Also italic isn't a color, so color is misleading. I think that means you can change the class name to be about Formatting too.

Reply:
Turns out `format` is a key word, so I've updated it to be `text_format` 👍